content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,15 +9,46 @@ menu:
     parent: About the project
 ---
 
-## v1.9.4 []
+## v1.9.4 [TK]
 
 ### Features
-
-- Added `memUsage` metrics to `/debug/vars` endpoint.
-  Measures memory usage in bytes across all subscriptions.
+- Add more options for [restoring](/enterprise_influxdb/v1.9/tools/influxd-ctl/#restore) Enterprise databases.
+  Users can now:
+  restore data with a new retention policy into an existing database,
+  override the duration of a retention policy while restoring, and
+  specify a destination shard when restoring a specific shard. <!-- https://github.com/influxdata/plutonium/pull/3608 -->
+- Add `memUsage` metrics to `/debug/vars` endpoint
+  to measure memory usage in bytes across all subscriptions.<!-- https://github.com/influxdata/influxdb/pull/22020 -->
+- Ensure `SHOW TAG VALUES` produces results from one specific retention policy. <!-- https://github.com/influxdata/influxdb/pull/22011 -->
 - Make `compact-full-write-cold-duration` apply to both TSM files and the TSI index.
+- Update Protocol Buffers library versions. <!-- https://github.com/influxdata/influxdb/pull/21890 -->
+- Update to Flux [0.124.0](/influxdb/v2.0/reference/release-notes/flux/#v01240-2021-08-03). <!-- https://github.com/influxdata/influxdb/pull/22102 -->
 
 ### Bug fixes
+- Require database authorization to see continuous queries. <!-- https://github.com/influxdata/influxdb/pull/22289 -->
```

Comment:
maybe 'Forbid users without read authorization on a database to see continuous queries linked to that database' or similar?